### ISSUE TITLE
Bug Report: I can not access the debug menu on the Newgrounds version

### ISSUE THREAD

**Issue by nothatguy64** [FunkinCrew/Funkin#2620]
### If you are playing the game in a browser, what site are you playing it from?

- [X] [Newgrounds](https://www.newgrounds.com/portal/view/770371)
- [ ] [Itch.io](https://ninja-muffin24.itch.io/funkin)? Specify below
- 
  - [ ] Windows
- 
  - [ ] Mac
- 
  - [ ] Linux

### If you are playing the game in a browser, what browser are you using?

- [X] Google Chrome (or chomium based like Brave, vivaldi, MS Edge)
- [ ] Firefox
- [ ] Safari

## What version of the game are you using? Look in the bottom left corner of the main menu. (ex: 0.2.7, 0.2.1, shit like that)

I am currently Using version 0.3.3

## Have you identified any steps to reproduce the bug? If so, please describe them below in as much detail as possible. Use images if possible.

Maybe this is an issue with the preferences menu, given that you have to turn this on in order to access it. Either that, or it's an issue with keybinds because I rewire them, but it doesn't work. Maybe yall's just added the option, but did not allow it to function. Similar to the naughtiness option, where it doesn't do anything.

## Please describe your issue. Provide extensive detail and images if possible.

I keep rewiring my controls, however when I click them in the menu, or in a song, they don't work. At all. Even with the preference for the menu to show, it simply does not work. No matter what you rewire what to. The chart, the stage, or the menu.

## If you're game is FROZEN and you're playing a web version, press F12 to open up browser dev window, and go to console, and copy-paste whatever red error you're getting

**Comment by GemFin** [comment]
last i heard, the newgrounds version doesnt have debug support due to issues

**Comment by NotHyper-474** [comment]
> last i heard, the newgrounds version doesnt have debug support due to issues

Yeah: https://twitter.com/ninja_muffin99/status/1785492222898577527

**Comment by mahikotahi** [comment]
web version doesn't have debug

**Comment by Hundrec** [comment]
Though the HTML5 build's debug menu is a bit bugged, we can close this issue.

![image](https://github.com/user-attachments/assets/f8cd7e9d-d8bd-43f5-94ce-bb34135cbb04)
